listing-rewards/program/src/rewardable_collection/close/mod.rs

```diff
@@ -0,0 +1,81 @@
+use anchor_lang::prelude::*;
```

Comment:
Let's call this delete because it is my intention to have a close account that zeros out the account and returns funds to the authority.